app/src/androidTest/java/org/kiwix/kiwixmobile/intro/IntroRobot.kt

```diff
@@ -49,8 +49,8 @@ class IntroRobot : BaseRobot() {
     }
   }
 
-  infix fun clickGetStarted(func: MainRobot.() -> Unit): MainRobot {
+  infix fun clickGetStarted(func: KiwixMainRobot.() -> Unit): KiwixMainRobot {
     clickOn(getStarted)
-    return main(func)
+    return kiwixMainRobo(func)
```

Comment:
can we change this function back to `main` to match the convention of the other robots?

I am also in a bit of a quandry about what it should be called, mainActivity used to be quite a fixed screen and now main is the host of all screens? It is probably fine to keep calling it main but it is a bit disingenuous.... maybe `topLevelDestination`?

Reply:
What about changing robot's name to `kiwixMain()` instead `main()`(as we are prefixing Kiwix to all the Main components almost everywhere(so far what I've explored till now) in the module), and don't we any other alternatives for the file name, `topLevelDestination` seems less discoverable enough for beginner contributors and a file with previously mentioned name would look nice inside a package named `main`, so for that do we need to change package name as well?

One point I have to support the `kiwixMain` and `KiwixMainActivityTest` is: even though an activity is (mostly)limited to a single screen only and in its automated test, we test each and every component of it being displayed on the screen; so we can use the similar analogy/reasoning for this test as well, as we've (I think) exhaustively tested the KiwixMainAcitivity

Reply:
`topLevelDestinations` makes most sense to me as robots are UI conscious components, we want to provide an entry point for interacting with the UI on screen aka the nav view and the visibility of this view is goverened by our notion of topLevelDestinations. Exposing these functions eg `clickOnReader/Download/Library` on main is erroneous as they are not always presented and therefore could be misused causing a test error.

We could have a `mainRobot` but it would match the actual hierarchy of the application and `topLevelDestinations` would be a robot under it